starboard/nplb/socket_create_test.cc

```diff
@@ -80,29 +82,7 @@ TEST_P(SbSocketCreateTest, ManyTcpAtOnce) {
   }
 }
 
-#if SB_HAS(IPV6)
```

Comment:
Why did you change the existing test?

Reply:
It is a mistake. The change will be removed.